### DISCUSSION TITLE
League rank doesnt show in overall hiscore.

### DISCUSSION THREAD

**Discussion by Famezxl** [runelite/runelite#17952] [Ideas]
Just a pet-peeve but the leagues rank for the previous season doesnt show unless you go into the leagues specific hiscores, any fix?

**Comment by Nightfirecat** [comment]
League stats are not listed on the main hiscores, so there's no workaround on our side for this. Please direct this request to Jagex.

**Reply by Famezxl** [reply]
Theres no way to get that data imported to the "overall" tab in runelite?

**Reply by Nightfirecat** [reply]
That tab loads data from the main game hiscores, so no
